sources/packages/backend/apps/api/src/services/student/student-information.service.ts

```diff
@@ -31,6 +46,89 @@ export class StudentInformationService {
     });
   }
 
+  /**
+   * Get student applications.
+   * @param studentId student id.
+   * @returns applications.
+   */
+  async getStudentApplications(
+    studentId: number,
+  ): Promise<ApplicationDetail[]> {
+    // Get the SQL to get last 3 active program years including the current program year.
+    const programYearQuery = this.programYearRepo
+      .createQueryBuilder("programYear")
+      .select(["programYear.id"])
+      .where("programYear.active = true")
+      .andWhere("programYear.startDate <= NOW()")
+      .orderBy("programYear.startDate", "DESC")
+      .limit(3)
```

Comment:
Please move the 3 as a const, for instance, "MAX_PAST_PROGRAM_YEARS" or similar.